py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -95,6 +95,8 @@ def __call__(self, ignored):
 
 
 def _element_if_visible(element, visibility=True):
+    if isinstance(element, str) or isinstance(element, dict):
+           raise StaleElementReferenceException("Invalid locator")
```

Comment:
You wrote `expected_conditions.invisibility_of_element_located((By.CSS_SELECTOR, elem))`
Is `elem` a string? 
If it's an element you should use `visibility_of(element)` with `until_not` or `staleness_of`.